asset-transfer-secured-agreement/application-gateway-typescript/src/utils.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import crpto from 'crypto';
+
+export const RED = '\x1b[31m\n';
+export const GREEN = '\x1b[32m\n';
+export const RESET = '\x1b[0m';
+
+export interface AssetJSON {
+    objectType: string;
+    assetID: string;
+    ownerOrg: string;
+    publicDescription: string;
+}
+
+export interface AssetPropertiesJSON {
+    objectType: string;
+    assetID: string;
+    color: string;
+    size: number;
+    salt: string;
+}
+
+export interface AssetPriceJSON {
+    assetID: string;
+    price: number;
+    tradeID: string;
+}
```

Comment:
These JSON interfaces are only used internally by the ContractWrapper class. No other code uses or needs to know about them. They should be non-exported members of the contractWrapper module.